src/spyglass/behavior/moseq.py

```diff
@@ -0,0 +1,360 @@
+import hashlib
+import os
+from pathlib import Path
+
+import datajoint as dj
+import keypoint_moseq as kpms
+
+from spyglass.common import AnalysisNwbfile
+from spyglass.position.position_merge import PoseOutput
+from spyglass.utils import SpyglassMixin
+
+from .core import PoseGroup, format_dataset_for_moseq, results_to_df
+
+schema = dj.schema("moseq_v1")
+
+
+@schema
+class MoseqModelParams(SpyglassMixin, dj.Manual):
```

Comment:
I don't love the convention of table names prefixed by the software dependency (`DLCModelX`, `MoseqModelX`). It permits

```python
from spyglass.behavior.moseq import *
from spyglass.position.whatever import *

DLCModel()
MoseqModel()
```

I'd prefer to encourage

```python
from spyglass.behavior import moseq
from spyglass.position.whatever import dlc_model as dlc

moseq.Model()
dlc.Model()
```

but it's hard to shift to the latter when we already have the former